src/main/java/com/microsoft/sqlserver/jdbc/Parameter.java

```diff
@@ -531,8 +532,14 @@ private void setTypeDefinition(DTV dtv) {
                         if (userProvidesPrecision) {
                             param.typeDefinition = "decimal(" + valueLength + "," + scale + ")";
```

Comment:
I know this isn't in the PR but why is "decimal" hardcoded instead of using SSType like the others?

Reply:
Not sure, it may have been easier to concatenate that way initially. I can swap it out for SSType.DECIMAL.toString() (there are only 8 occurrences like the above).